### DISCUSSION TITLE
Is it possible to control Relay by HTTP Commands ?

### DISCUSSION THREAD

**Discussion by Rimula-X** [easytarget/esp32-cam-webserver#283] [Common Issues]
Due to I have a opencv project by 
1. pc capture image from esp32-cam (http://ip-address/capture)
2. process image with face recognition method using python
3. if the face match with data, open the door (work with relay 1-4), else turn on alert ( work on relay 5-8)
4. in terms of physical connection, I connect 8 relays with i/o expansion to ESP32-cam GPIO (due to less pin) and use pc to control both camera and relay over wifi. 
5. now I can capture image by HTTP Commands, but I have no idea How to turn on/off relay from pc. 
As I know http://ip-address/control? should be modified to control GPIO pin if it connect directly, 
but I still have no idea how to use HTTP Commands to control relay via i/o expansion, please advice.

**Comment by israelcasado** [comment]
Did you find out how to do it? 
 You have several options to do this: from the ESP32 itself or from the PC or device that is running the Python script.

**Comment by easytarget** [comment]
Sorry; I am not doing any further development on this project. it is old code and somewhat obsolete. I am currently working on retiring it; I will try to link to a more actively updated fork when I do.
